client/components/AlbumDisplay/index.tsx

```diff
@@ -108,26 +108,20 @@ const AlbumDisplay: React.FC<AlbumDisplayProps> = (
             <div className={style.rating} onClick={(e) => e.preventDefault()}>
                 <Rating value={props.album.rating} fav={props.album.flag} />
             </div>
-            <Link to={`/album/${props.album.id}`} className={style.details}>
+            <div className={style.details}>
```

Comment:
Is the entire thing not clickable anymore? I would have preferred it remain fully clickable. Up to debate though.

Reply:
Understandable, though I found myself misclicking fairly often and the album title changing color even though other links were hovered on felt unclear.

There are still two ways to get to the album, clicking the album title or the View Album link in the hover actions.